controller/src/main/java/io/pravega/controller/server/eventProcessor/ScaleOpEvent.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server.eventProcessor;
+
+import io.pravega.shared.controller.event.ControllerEvent;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.AbstractMap;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+public class ScaleOpEvent implements ControllerEvent {
```

Comment:
There is autoScale event for all auto scale requests coming from SSS.. 
so to disambiguate we call this scale op event.